config/passport.js

```diff
@@ -60,7 +60,7 @@ module.exports = function(passport) {
                     return done(null, user); // user found, return that user
                 } else {
                     // if there is no user found with that facebook id, create them
-                    var newUser            = new User();
+                    const newUser            = new User();
```

Comment:
True. thanks @vegeta1509 . @VibhorCodecianGupta go through the code once more and recheck this type of mistakes. Also, i see changes in angular files so revert them as well